### ISSUE TITLE
Julie's Macbook setup 

### ISSUE THREAD

**Issue by jules32** [Openscapes/teaching-learning-resources#12]
Had a new logicboard and whole harddrive was reformatted, so starting afresh: 

## System preferences

trackpad 
- Point & Click: tap to click > check
- Scroll & Zoom: scroll direction natural > uncheck

general 
- show scroll bars > always

keyboard
- keyboard > Touchbar shows expanded control strip > check, then Customize Control Strip: 
    - https://support.apple.com/guide/mac-help/customize-the-touch-bar-mchl5a63b060/mac
- shortcuts > screenshots > copy picture of selected area to the clipboard > command 4

doc & menu bar
- size: small
- magnification: middle
- automatically hide & show the doc > check

notifications
- turn on do not disturb: when mirroring to projectors > check

## Safari 

- show full url, see #11

**Comment by jules32** [comment]
## R, RStudio, Git setup 

Follow https://ucsb-meds.github.io/meds-install-mac.html. Exceptions:

- I installed the RStudio Daily since I'm working with Quarto: https://dailies.rstudio.com/
- Didn't install Anaconda
- RStudio suggested installing XCode, said yes.
- Git setup: in addition to storing my GitHub PAT, I set up my SSH keys: https://happygitwithr.com/ssh-keys.html
    - I set a passphrase (for the first time). 
    - To Create a file `~/.ssh/config` I typed in the command line: 
```
cd .ssh
touch config
nano config 
< paste>
 Host *
 AddKeysToAgent yes
 UseKeychain yes
<control X, yes, enter >
```

Quarto Install
- https://quarto.org/docs/getting-started/installation.html
- `quarto serve` - woop woop
- `quarto render index.qmd` - yes install all required packages

**Comment by jules32** [comment]
## Garrick's setup
https://twitter.com/grrrck/status/1333986413906436098

I didn't do everything listed here, but yes to these: 
- https://addons.mozilla.org/en-US/firefox/addon/copy-as-markdown/ **Firefox add-on**

**Comment by jules32** [comment]
## 1 Password

## Zotero

- https://www.zotero.org/
- Login from browser
- Download desktop app and Zotero Connector to Firefox **Firefox add-on**
- Desktop app: login, approve syncing, press the sync green circle-arrow to start syncing

## Slack

- https://slack.com/downloads/mac
- Login to connect existing workspaces to Slack

## Zoom
- https://zoom.us/download
- (Julie sign in with SSO to UCSB)

**Comment by jules32** [comment]
# [Dragon Software (by Nuance)](https://www.nuance.com/dragon.html) setup

I bought the Professional Individual account, learning that this can be run on a Mac using a virtual machine ([blog](https://machow2.com/best-dictation-software-mac/))

From a Mac, requires [Parallels](https://www.parallels.com) and Windows. Then, move the .exe to the Local Disk C: drive (and I saved it there as well). This followed [Claire G's advice on the Parallels Forum](https://forum.parallels.com/threads/problems-with-nuance-dragon-download-manager.350659/) about the Download Manager hanging for hours and hours when I double-clicked the .exe from the Mac's Downloads folder

(update: I returned it and started investing the time into physical therapy and strength building for my arms/RSI)

**Comment by jules32** [comment]
## Mac OS Sonoma 14.2.1 - need to install XCode

**TL;DR**: 

`xcode-select --install`

## The story

Last week I upgraded to OS Sonoma. Today when I opened RStudio, there was no "Git" tab. In the terminal, `git status` errored: 

```
git status
xcrun: error: invalid active developer path (/Library/Developer/CommandLineTools), missing xcrun at: /Library/Developer/CommandLineTools/usr/bin/xcrun
```

I googled the error message, found the solution here: <https://forums.developer.apple.com/forums/thread/673827>

So in the terminal, I typed: `xcode-select --install`. It cryptically returned `xcode-select: note: install requested for command line developer tools`.  I typed it again. Same cryptic message. Using the keyboard shortcut `command-tab` I noticed an unfamiliar application was open, some kind of installer. I looked it it - **it was a dialog box** that I needed to click in order for this to fully install. 

<img width="948" alt="Screenshot 2024-01-21 at 13 25 59" src="https://github.com/Openscapes/teaching-learning-resources/assets/5891909/a22d192b-0919-44b5-b74a-0ea91c58ca2d">

It said it would take 47 hours to download. Then corrected to 9 minutes. When it finished, I quit RStudio, and when reopened, I had the "Git" tabs again. 😎
